index.bs

```diff
@@ -1669,20 +1672,20 @@ guarantee.
 </div>
 
 
-# Accessing the Origin Private File System # {#sandboxed-filesystem}
+# Accessing the Bucket File System # {#sandboxed-filesystem}
 
-The <dfn export>origin private file system</dfn> is a [=storage endpoint=] whose
+The <dfn export>bucket file system</dfn> is a [=storage endpoint=] whose
```

Comment:
We should keep the existing ID. (For the other `<dfn>` change I don't think it matters since we introduced it two days ago, but this is much more likely to be referenced already.)

And FWIW, in my opinion it's fine if the ID has a different name from what it identifies. It nicely shows the specification has evolved.

Reply:
Done. I was ready to update all the refs anyways (see https://github.com/WICG/file-system-access/pull/416) but it is nice to see the evolution :)